### PR TITLE
Database docs improvements

### PR DESCRIPTION
**Pull request by bilalashraf123**
Documentation improvements related to database configurations.

This pull request closes the following issues:
- https://github.com/FlowCrypt/flowcrypt-docs/issues/110
- https://github.com/FlowCrypt/flowcrypt-docs/issues/112
- https://github.com/FlowCrypt/flowcrypt-docs/issues/119

### CODE REVIEW

File: _pages/technical/enterprise/configuration/postgresql.md [hunk 1]
```diff
@@ -4,65 +4,121 @@ categories: page
...
+GRANT SELECT, UPDATE ON ALL SEQUENCES IN SCHEMA ekm TO ekm_rw;
+ALTER DEFAULT PRIVILEGES IN SCHEMA ekm GRANT SELECT, INSERT, UPDATE, DELETE ON TABLES TO ekm_rw;
+ALTER DEFAULT PRIVILEGES IN SCHEMA ekm GRANT SELECT, UPDATE ON SEQUENCES TO ekm_rw;
 ```
 
+### Additional steps for TLS Client Certificate based authentication configuration
+
+For database server keys, follow PostgreSQL documentation. For client keys, create / export a private key as pkcs8 - not encrypted - DER format:
+
+ - openssl: You can convert a PEM key to pkcs8 using openssl pkcs8 -topk8 -inform PEM -outform DER -in ekm.key -out ekm.pk8 -nocrypt
```
Comment (tomholub) [inline _pages/technical/enterprise/configuration/postgresql.md:89]: Missing backticks for the command to format it as code

File: _pages/technical/enterprise/configuration/postgresql.md [hunk 2]
```diff
@@ -4,65 +4,121 @@ categories: page
...
+| Property | Example | Description |
+|:---------|:---------|:------------ |
+| `store.type` | depends on service | Use `EkmPostgresStore` for EKM, or `WkdPostgresStore` for WKD |
+| `store.postgres.host` | `10.0.0.5` | Postgres database hostname or IP address |
+| `store.postgres.host.readonly` | `10.0.0.6` | Optional - separate hostname or IP address for readonly requests |
+| `store.postgres.port` | `5432` | Postgres database port number |
+| `store.postgres.database` | `db_ekm` | Postgres database name. You can choose any. Recommend `db_ekm` or `db_wkd` |
+| `store.postgres.connections` | `12` | Postgres database connections (recommend 3 per cpu core) |
+| `store.postgres.update.schema` | `true` | If schema should be created / updated when the service starts. |
+| `store.postgres.username` | `user_ekm` | Postgres database user name. You can use any. Recommend `user_ekm` or `user_wkd` |
```
Comment (tomholub) [inline _pages/technical/enterprise/configuration/postgresql.md:107]: I think it's missing a property to choose schema, it was something like `store.postgres.schema` but please cross-check.

**Review by tomholub — COMMENTED** [review]
Some things to update, else looks good - it's a clarity improvement.

File: _pages/technical/enterprise/configuration/postgresql.md (hunk 1 above)
Comment (bilalashraf123) [reply _pages/technical/enterprise/configuration/postgresql.md]: Done.

File: _pages/technical/enterprise/configuration/postgresql.md (hunk 2 above)
Comment (bilalashraf123) [reply _pages/technical/enterprise/configuration/postgresql.md]: Done.

**Review by tomholub — APPROVED** [review]

**Pull request merged**